sale_isolated_quotation/models/sale.py

```diff
@@ -11,19 +11,19 @@ class SaleOrder(models.Model):
         string='Is Order',
         readonly=True,
         index=True,
-        default=lambda self: self._context.get('is_order', False),
+        default=lambda self: self.env.context.get('is_order'),
```

Comment:
@bealdav Maybe this should be avoided using the '_default_<field>' value in context behaviour ?

Reply:
Now removed as requested